app/api/job/createJob.ts

```diff
@@ -194,31 +188,42 @@ export async function createJobAppointment({
     };
 
     logger.info({
-      message: "Creating job",
       jobData: jobData,
-    });
+    }, "createJobAppointment: Creating job");
     const jobResponse = await serviceTitanClient.jpm.JobsService.jobsCreate({
         tenant: tenantId,
         requestBody: jobData
     });
     logger.info({
-      message: "Job created",
       jobId: jobResponse.id,
-    });
+    }, "createJobAppointment: Job created");
 
     // Check and assign managed tech if needed
-    await checkAndAssignManagedTech(jobResponse, Number(technicianId));
+    try {
+      await checkAndAssignManagedTech(jobResponse, Number(technicianId));
+    } catch (error) {
+      logger.error({ err: error }, "createJobAppointment: Error checking and assigning managed tech");
+    }
+
+    try {
+      await updateInvoiceAndPayment(jobResponse, serviceTitanClient, Number(technicianId));
+    } catch (error) {
+      logger.error({ err: error }, "createJobAppointment: Error updating invoice and payment");
+    }
```

Comment:
This is the main fix.

Previously, if we failed to create the invoice and payment, the user would get an error message saying "payment succeeded but booking failed".

That error message was not even true since the booking WAS created but updating other ST records is what failed. So we just catch this error now and allow the method to return successfully.